starter-code/iteration-1/main.ts

```diff
@@ -1,11 +1,27 @@
 // Iteration 1
 // 1. Create an array of strings
+let TodoList: Array<string> = [];
 
 // 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
+function addTask(task:string) {
```

Comment:
you could also add the number type: 
`function addTask (task: string): number {
`